slick/src/main/scala/slick/jdbc/SQLiteProfile.scala

```diff
@@ -97,9 +97,10 @@ trait SQLiteProfile extends JdbcProfile {
   class ModelBuilder(mTables: Seq[MTable], ignoreInvalidDefaults: Boolean)(implicit ec: ExecutionContext) extends JdbcModelBuilder(mTables, ignoreInvalidDefaults) {
     override def createColumnBuilder(tableBuilder: TableBuilder, meta: MColumn): ColumnBuilder = new ColumnBuilder(tableBuilder, meta) {
       /** Regex matcher to extract name and length out of a db type name with length ascription */
-      final val TypePattern = "^([A-Z]+)(\\(([0-9]+)\\))?$".r
+      final val TypePattern = """^([A-Z]+)(|\(([0-9]+)(|,([0-9]+))\))?$""".r
```

Comment:
https://regex101.com/r/lY3kG4/2 <-- this site does show match information in right site, showing different captured groups, best online regexp testing site that I know, also provides explanation. Only thing for me it hasn't worked with is named capture groups with duplicate names